capture-sdk/sdk/src/main/res/layout/gc_detection_error_layout.xml

```diff
@@ -0,0 +1,30 @@
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/gc_medium"
+    android:background="@drawable/gc_detection_error_background"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <TextView
```

Comment:
[Request] Please use `style="@style/GiniCaptureTheme.Typography.Body2"` for the text.